test/runtests.jl

```diff
@@ -1,15 +1,15 @@
 # Adapted from scikit-learn
 # Copyright (c) 2007–2016 The scikit-learn developers.
 
+import PyCall
+
 #use non-mkl versions of python packages (to allow MacOS tests pass)
 @static if Sys.isapple()
-    using Conda
-    Conda.add("nomkl")
-    Conda.rm("mkl")
+    PyCall.Conda.add("nomkl")
+    PyCall.Conda.rm("mkl")
```

Comment:
Yeah, this looks wrong, it just reinforces that we should have `Conda` in `Project.toml` as you did it first, so that we can `import Conda` normally.

Reply:
Could you please readd Conda to Project.toml and change the calls here to `Conda.add/rm`? Conda is installed anyway via PyCall, and it's just cleaner to directly call it/require it.